Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -583,7 +597,6 @@ void RemovePage(Page page)
 
 			// Go ahead and take care of the fragment bookkeeping for the page being removed
 			FragmentTransaction transaction = FragmentManager.BeginTransaction();
-			transaction.DisallowAddToBackStack();
```

Comment:
This method call was removed in #919 for master-detail and navigation pages. Looks like this was missed. Let me know if this needs to go in another PR.

Reply:
This is fine - we should not be using DisallowAddToBackStack anywhere.